README.md

```diff
@@ -1 +1 @@
-# hello-php
```

Comment:
Seems ok. Can you add a comment explaining what this changes will do ?
don't want the app to fail with exceptions after deployment.